assets/src/blocks/Carouselheader/CarouselHeaderBlock.js

```diff
@@ -150,7 +150,20 @@ export class CarouselHeaderBlock {
               link_url: '',
               link_url_new_tab: false,
             }
-          ]
+          ],
+          validation: slides => {
+            let isSlidesValid = true;
```

Comment:
Maybe a bit more straightforward here if you use `invalidSlides = slides.filter(slide => slide.image === null` and  `const isValid = invalidSlides.length > 0`, `const messages = invalidSlides.map(slide => ...)`, and then `return {isValid, messsages}`, as you wouldn't need both these variables.